### PR TITLE
Site design tweaks

### PR DESCRIPTION
**Pull request by JoanaMMoreira**
- hide scrollbar on docs side navigation
- remove hover background colour on navbar items
- add base root text styles to match Docusaurus
- amend features list icon colour
- remove hover background colour on hamburger menu button
- increase spacing between GitHub logo and label on mobile menu
- remove hover background colour on mobile menu close button
- amend mobile menu items font size
- change background image size on docs to make all pages consistent
- remove duplicate utils function
- update code blocks font family

### CODE REVIEW

**Review by james-nash — APPROVED** [review]
Nicely done! 👏

**Review by joshwooding — APPROVED** [review]

File: mosaic-site/src/_index/features/Features.module.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
+.listItem > svg {
+  --saltIcon-color: var(--salt-actionable-primary-background-active);
```
Comment (alycrys) [inline mosaic-site/src/_index/features/Features.module.css:2]: Just checking, shouldn't this be 'foreground'?
CC @libertymayc

File: mosaic-site/src/css/global/code.css [hunk 2]
```diff
@@ -10,7 +10,7 @@ pre {
   --pre-color: #393a34;
   --code-font-size: 90%;
   --pre-line-height: 1.45;
-  --font-family-monospace: SFMono-Regular, Menlo;
+  --font-family-monospace: var(--salt-text-code-fontFamily);
```
Comment (alycrys) [inline mosaic-site/src/css/global/code.css:13]: this var is only being used in one place. Do you think it makes sense to remove it and use `--salt-text-code-fontFamily` directly?

Comment (JoanaMMoreira) [reply mosaic-site/src/css/global/code.css:13]: I think this is just to help simply the code on line 72 ` font: var(--code-font-size) / var(--pre-line-height) var(--font-family-monospace);`

File: mosaic-site/src/_index/features/Features.module.css (hunk 1 above)
Comment (alycrys) [reply mosaic-site/src/_index/features/Features.module.css:2]: could be the right choice, was just checking. If I remember well, @pseys mentioned we should use foreground for SVGs, but he might be referring only to Salt components

File: mosaic-site/src/css/global/code.css (hunk 2 above)
Comment (alycrys) [reply mosaic-site/src/css/global/code.css:13]: oh, I thought those vars were created because they had custom values.

**Review by alycrys — APPROVED** [review]

**Pull request merged**